experimental/streaming/exporter/reader/reader.go

```diff
@@ -265,22 +222,32 @@ func (ro *readerObserver) orderedObserve(event observer.Event) {
 			read.SpanContext = span.spanContext
 		}
 
-	case observer.RECORD_STATS:
-		read.Type = RECORD_STATS
+	case observer.UPDATE_METRIC:
+		read.Type = event.Type
 
-		_, span := ro.readScope(event.Scope)
-		if span != nil {
-			read.SpanContext = span.spanContext
+		if event.Context != nil {
+			span := trace.CurrentSpan(event.Context)
+			if span != nil {
```

Comment:
Should we warn about /ignore the event if the span is `nil`? The spanlog reporter is going to panic in this situation…

Reply:
We should treat the metric event as though it has no span context, otherwise it should be OK